nautobot_golden_config/nornir_plays/config_compliance.py

```diff
@@ -134,6 +134,10 @@ def run_compliance(  # pylint: disable=too-many-arguments,too-many-locals
     obj = task.host.data["obj"]
     settings = device_to_settings_map[obj.id]
 
+    if not settings.compliance_enabled:
```

Comment:
Same as previous. thinking the proper settings verification needs to either happen in the existing verify_settings method and/or earlier in the process.